extensions/one-time-password/src/config.ts

```diff
@@ -0,0 +1,15 @@
+const theme = {
+  grey: '#909090',
+  green: '#288056',
+  red: '#802828',
```

Comment:
What about using `Color.Red`, `Color.Green`, `Color.SecondaryText` instead here

Reply:
green and red are not used right now but for all 3 the idea was to use something less bright no not grab too much attention.

Reply:
It is possible to create your own themes in Raycast PRO. If you don't consider this, it might conflict with the theme

Reply:
I see. I removed the custom colors